docs/02-get-started/03-deploy-expose-microservice.md

```diff
@@ -0,0 +1,242 @@
+```bash
+kubectl get service orders-service -o=jsonpath="{.metadata.uid}"
+```
+
+The operation was successful if the command returns the **uid** of your Service.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+As you've already [created the Kubernetes Service with the microservice](#deploy-the-microservice), skip this part.
+
+<!--
+//TODO: Functionality not added yet. Check with Hasselhoffs in a while.
+If you created the Service at the previous step while creating the Deployment, skip this section. Otherwise, you must now create the Service.
+
+1. From the left navigation, go to **Services**.
+2. Click on **Create Service +**.
+3. ...
+
+The operation was successful if ... .
+--->
+  </details>
+</div>
+
+## Expose the microservice
+
+We have created the Service. Let's now expose it outside the cluster.
+
+> **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always [secure the workload](../03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md) you expose.
+
+To expose our microservice, we must create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) CR for it, just like when we [exposed our Function](02-deploy-expose-function.md#expose-the-function).
+
+<div tabs name="Expose the microservice" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  name: orders-service
+  namespace: default
+  labels:
+#    app: orders-service
+#    example: orders-service
+spec:
+  service:
+    host: orders-service.$CLUSTER_DOMAIN
+    name: orders-service
+    port: 80
+  gateway: kyma-gateway.kyma-system.svc.cluster.local
+  rules:
+    - path: /.*
+      methods: ["GET","POST"]
+      accessStrategies:
+        - handler: noop
+      mutators: []
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In your Services's view, click on **Expose Service +**.
+2. Provide the **Name** (`hello-world`) and **Hostname** (`hello-world`) and click **Create**.
+
+> **NOTE:** Alternatively, from the left navigation go to **APIRules**, click on **Create apirules +**, and continue with step 2, selecting the appropriate **Service** from the dropdown menu.
+  </details>
+</div>
+
+### Verify the microservice exposure
+
+Now let's check that the microservice has been exposed successfully.
+
+<div tabs name="Verify microservice exposure" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+curl https://orders-service.$CLUSTER_DOMAIN/orders
+```
+
+The operation was successful if the command returns the (possibly empty `[]`) list of orders.
+
```

Comment:
again the 443 error :(
![image](https://user-images.githubusercontent.com/76950046/133645245-8303a7d6-3c7b-4ead-933b-0e0e1ce9817d.png)

Reply:
The local setup isn't working right now.